packages/paste-theme/src/utils/ReactHelmetOverride.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import {Helmet as ReactHelmet, HelmetProps as ReactHelmetProps} from 'react-helmet';
+
+interface HelmetProps extends ReactHelmetProps, React.PropsWithChildren {}
+
+export const Helmet = (props: HelmetProps) => <ReactHelmet {...props} />;
```

Comment:
this is no longer required. I have already fixed the Helmet issue inside the base branch

Reply:
amazing, thank you! reverting 🧨